XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Simulation/SimulationTimeStampStopWatch.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Diagnostics;
+
+namespace XRTK.Providers.Controllers.Simulation
+{
+    public sealed class SimulationTimeStampStopWatch
```

Comment:
Do we really need this class to be specific to Simulation?

I'm thinking this could be a more generic named class and live in Utilities or something.

Reply:
No we don't. But most likely never to be used anywhere else. I'll move it.